src/titiler/core/titiler/core/factory.py

```diff
@@ -852,6 +874,7 @@ def tile(
             """Create map tile from a dataset."""
             tms = self.supported_tms.get(tileMatrixSetId)
             with rasterio.Env(**env):
+                logger.info(f"opening data with reader: {self.reader}")
```

Comment:
> I am not sure how useful those 'opening dataset with reader: <Reader>' messages will be so we could take them out, but I thought adding some more visibility into the specific reader class that is getting used could be helpful in some contexts.

Yeah, we can keep them (maybe remove `with reader: {self.reader}`), I'll be fine to add more logger statement for the `reading` part and rendering, it will be a lot of logging step to add but 🤷

Reply:
I like having the `reader` class logged because it can help a user understand how the tiles are actually being generated. You can get this info from a traceback but if you don't hit an error than it is not always clear!

I could add `reader`/`backend` attributes to telemetry traces instead if you would rather leave the reader class out of the actual log messages.